pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -1022,9 +1037,42 @@ public void rinkebyValuesCanBeOverridden() throws Exception {
     assertThat(networkArg.getValue().getNetworkId()).isEqualTo(1);
   }
 
+  @Test
+  public void cliOptionsNotShownWhenInDockerContainer() {
+    System.setProperty("pantheon.docker", "true");
+
+    parseCommand("--help");
+
+    verifyZeroInteractions(mockRunnerBuilder);
+
+    assertThat(commandOutput.toString()).doesNotContain("--config");
+    assertThat(commandOutput.toString()).doesNotContain("--datadir");
+    assertThat(commandOutput.toString()).doesNotContain("--genesis");
+    assertThat(commandErrorOutput.toString()).isEmpty();
+
+    System.setProperty("pantheon.docker", "false");
+
+    parseCommand("--help");
+
+    verifyZeroInteractions(mockRunnerBuilder);
+
+    assertThat(commandOutput.toString()).contains("--config");
+    assertThat(commandOutput.toString()).contains("--datadir");
+    assertThat(commandOutput.toString()).contains("--genesis");
+    assertThat(commandErrorOutput.toString()).isEmpty();
+  }
+
   private Path createFakeGenesisFile() throws IOException {
     final Path genesisFile = Files.createTempFile("genesisFile", "");
     Files.write(genesisFile, "genesis_config".getBytes(UTF_8));
     return genesisFile;
   }
+
+  private Boolean isDockerInstantiation() {
```

Comment:
I had that originally, but figured (for testing) it'd be more useful to test that each exclusion parameter (isDocker in this case) is being individually tested.  

In future I can a situation where we'll have multiple parameters that exclude different options (with some parameters excluding the same options); and we need to test each individually to ensure that each individual parameter is excluding the options we expect.

In PantheonCommand.java I checked the inverse condition `isFullInstantiation()` because it felt more natural in that context, as it's (currently) determining whether we want to include **all** the options, but that conflicts a bit with my previous reasoning...  🤔 

Thoughts?